### PR TITLE
feat: update to expecto 10

### PR DESCRIPTION
**Pull request by SimonCropp**
BREAKING-CHANGE: major expecto update and drops support for older .NET framework versions.

### CODE REVIEW

**Comment by Alxandr** [comment]
I'm on vacation. Hopefully I can get back to this in about a week.

**Comment by Alxandr** [comment]
The build fails, probably cause you've dropped support for older .NET targets, and now refer to files that does not exist. Why was support for older frameworks dropped?

**Comment by SimonCropp** [comment]
expecto dropped support for older frameworks

File: src/YoloDev.Expecto.TestSdk/YoloDev.Expecto.TestSdk.fsproj [hunk 1]
```diff
@@ -20,9 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="build/netcoreapp3.1/YoloDev.Expecto.TestSdk.props" Pack="true" PackagePath="build\netcoreapp3.1\" />
-    <None Include="build/netcoreapp3.1/_._" Pack="true" PackagePath="lib\netcoreapp3.1\" Visible="false" />
-    <None Include="build/net5.0/_._" Pack="true" PackagePath="lib\net5.0\" Visible="false" />
+    <None Include="build/net6.0/_._" Pack="true" PackagePath="lib\net6.0\" Visible="false" />
```
Comment (Alxandr) [inline src/YoloDev.Expecto.TestSdk/YoloDev.Expecto.TestSdk.fsproj:23]: Pretty sure this right here is your issue. The file `build/net6.0/_._` does not exist. The `net5.0` folder need to be renamed. And the `netcoreapp3.1` folder should probably be deleted.

**Comment by Alxandr** [comment]
The build passes, but I noticed it still installed .NET 3.1 and 5. Could you update the workspace files to no longer do that as well? Then I think this update would be good. Just have to make sure it triggers a new minor (not patch) release.

**Comment by SimonCropp** [comment]
done

**Review by Alxandr — APPROVED** [review]

**Comment by Alxandr** [comment]
If nothing comes up with 0.14 in about a month or so, I agree it's probably a good idea - but I do not want to release a 1.0 right after making a breaking change.

**Comment by xperiandri** [comment]
Why not? It is perfect reason to do that 🙂

**Comment by Alxandr** [comment]
Cause I want to make sure it's stable first.

**Pull request merged**